x/rollapp/keeper/msg_server.go

```diff
@@ -15,3 +20,40 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
 }
 
 var _ types.MsgServer = msgServer{}
+
+func (k msgServer) TriggerGenesisEvent(goCtx context.Context, msg *types.MsgRollappGenesisEvent) (*types.MsgRollappGenesisEventResponse, error) {
```

Comment:
I would move this method to a new file, as mentioned above: msg_server_trigger_genesis_event.go